### PR TITLE
Soen133 mobile dashboard

### PR DESCRIPTION
**Pull request by amqkv**
# Description

Implements/fixes [SOEN-113](https://soen390team8.atlassian.net/jira/software/projects/SOEN/boards/1/backlog?selectedIssue=SOEN-133)

- Fixed the mobile view for doctor's dashboard
- Added filtering options (reviewed and unreviewed)
- Patient card border is red if latest status is unreviewed and gray if reviewed

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Unit test

# Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
|Unreviewed image | Reviewed image|

### CODE REVIEW

**Review by jfv-2000 — APPROVED** [review]
C O O L W O R K N I C E A W E S O M E
squidward-dab

**Review by KelvinTrang — APPROVED** [review]
Goood work!

File: frontend/components/doctor/patient-charts-overview.tsx [hunk 1]
```diff
@@ -95,6 +95,7 @@ export default function PatientChartsOverview({ patientList }: { patientList: Pa
                     </TabPanels>
                 </Tabs>
             </Box>
-        </Flex>
+        </SimpleGrid>
+        // </Flex>
```
Comment (stevenvuong00) [inline frontend/components/doctor/patient-charts-overview.tsx]: remove comment ?

File: frontend/pages/doctor/patient-list-overview.tsx [hunk 2]
```diff
@@ -74,9 +75,23 @@ export default function DoctorDashboard({ patientList }: { patientList: Patient[
...
+        }
+        try {
+            return await fetch(serverURL + "/doctors/reviewPatient", {
+                method: "PATCH",
+                body: JSON.stringify({
+                    patientId: selectedPatient.patientId,
+                }),
+                headers: { "Content-Type": "application/json" },
+            });
+        } catch {}
```
Comment (stevenvuong00) [inline frontend/pages/doctor/patient-list-overview.tsx:94]: should this all be moved to another function?

**Review by stevenvuong00 — APPROVED** [review]
great work, looks clean, check out my comments

**Pull request merged**